build/scripts/build.ps1

```diff
@@ -26,6 +26,7 @@ param (
     [switch]$bootstrap = $false,
     [switch]$sign = $false,
     [switch]$pack = $false,
+    [switch]$updateLoc = $false,
```

Comment:
@tmeschter When is this command useful? No other repo has such a command.

Reply:
The .xlf files need to be kept up to date with the sources, but as you're making changes you probably don't want to deal with the 13+ updated files that show up just because you tweaked something in a .resx file. Instead, if you've touched a .resx file you run `build -updateLoc` and commit all the changed files once before pushing and creating a PR. If you forget to do this step then the PR build will complain.

Reply:
I'd rather deal with the updated files right away than forget to run the command and see my build fail in PR after half an hour. Having to deal with the xlf files has never been an issue for me in any of the repos I was updating resource files in.